src/main/resources/openapi/PrintProviderAPI.yaml

```diff
@@ -2,120 +2,149 @@ openapi: 3.0.0
 info:
   title: Print ProviderAPI
   version: '1.0'
-  description: API for Print providers to retrieve print requests and to send updates on print request statuses.
+  description: API Specification for sharing Models between EIP and Print suppliers that will be exchanged over SFTP.
```

Comment:
Given that our interface with print providers is SFTP, what "models" do we need to share with them?
What data structures do they need to know about?

Notwithstanding that (and this is something that @Neil-Massey and I are discussing at the moment in context of notifications API), openAPI is not the right tool for this.
openAPI spec is specifically for document REST APIs. Given that you have no `paths` elements, this spec is semantically invalid. Viewing it in swagger editor, or using a codegen tool will fail

Neil and I are considering this exact problem in notifications-api because we dont want the SQS models defined in the openAPI spec (because they are not related to REST APIs)

Reply:
Removed and moved to YAML json schema